### ISSUE TITLE
AIP 5: Token Burn

### ISSUE THREAD

**Issue by benevolentdictator** [airswap/airswap-aips#5]
Summary

This AIP counters AIP #2 , which proposes an airdrop for existing token holders, and instead, proposes a significant token burn to coincide with, or be announced with the relaunch. The concern is that an airdrop will result in significant sell pressure and destabilise the project at a critical juncture. 

Specification

Once accepted, a date will be announced to burn a yet to be determined number of undistributed tokens. All tokens that were designated to be included in the airdrop can instead be included in the token burn. 

**Voting.** The vote will be to execute a trial token burn of 100K AST. Vote on this proposal at https://vote.airswap.io/

Rationale

This proposal aims to ensure the long-term viability of the project and reward token holders, without creating potentially destabilising sell pressure.

Copyright

Copyright and related rights waived via CC0.

**Comment by Nobs0x** [comment]
I agree with this proposal 100%. As I stated in the other proposal (AIP-2) and the discord channel, this will lead to a price increase that will benefit more the long-term holders then an airdrop, leading to the massive token sale. 
The higher price will bring more new people to the project. Combining that with the liquidity mining and the fee-sharing incentives is a win-win. This proposal is fundamental to the project's stable future. 

If not all undistributed tokens are not burned, the amount that is not burned should go in a treasury that will function as security and guarantees that the project has stability.
For example, if we have 300M undistributed tokens, 70% can be burned, and the other 30% should go into the treasury.

**Comment by cryptoinvestingk-stack** [comment]
One of the most important thing for any project to work out is its token value accrual mechanism. Airdrop or LP detract value from the token by increasing 1) selling pressure and 2) supply. Both of which will lead to a negative expectation for rational actors. For example, CRV tokens. 

By contrast, burning a % of total tokens AS WELL AS sharing the fees generated with holders of the token will ensure 1) a clear value accrual mechanism and 2) positive expectation of token value. Thus creating a positive loop.

LP rewards should come from a treasury. Staked ASTs for liquidity provision should be have a % of the trading fee and the rest goes into the treasury. A long tailed distribution of tokens with perhaps vesting, say LP rewards tokens are distributed over 6 months, would align incentives even more

**Comment by AirswapWhale** [comment]
Glad to see this proposal. Airdrop provides zero value for the project.

I'm in favour of this burn. As mentioned on Discord, I will support a burn so that the 500 million token supply is reduced to 250 million. Meaning half of the supply is burned. Given the current circulating supply is around 170 million tokenw, we would still have an additional 80 million tokens for incentives like liquidity mining, staking or even building a treasury system later.

**Comment by jacktiansuperbway** [comment]
I also agree with this proposal 100%, this will indeed increase the total value of project and the confidence of the community members!

**Comment by dmosites** [comment]
This has been consolidated into #10.
